fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file "sagebio.jks" # Path to the json secret file - Follow https://github.com/fastlane/supply#setup to get one
```

Comment:
This file is committed as an encrypted file (sagebio.jks.enc) in the root of the repo.  there's a command to unencrypt this file in travis/build.sh then sagebio.jks is available to generate the the build that gets deployed to google play alpha.